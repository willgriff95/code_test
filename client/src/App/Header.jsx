import React from 'react'
import { Typography } from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';

const styles = theme => ({
    headerContainer: {
        background: '#0030ff',
    },
    headerText: {
      margin: '0 auto',
      width: '25%',
      color: 'white', 
      textAlign: 'left',
      letterSpacing: '15px',
      padding: '20px',
      lineHeight: '20px'
    }
  })

const Header = ({classes}) => (
    <div className={classes.headerContainer}>
        <Typography className={classes.headerText}>GENERAL  <br/>KNOWLEDGE_</Typography>
    </div>
);

export default withStyles(styles)(Header)