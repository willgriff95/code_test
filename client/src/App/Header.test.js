import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
})


it('render correctly Header component', () => {  
    const HeaderComponent = renderer.create(<Header />).toJSON();
    expect(HeaderComponent).toMatchSnapshot();
});