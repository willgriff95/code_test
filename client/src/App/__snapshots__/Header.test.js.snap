// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Header component 1`] = `
<div
  className="Header-headerContainer-39"
>
  <p
    className="MuiTypography-root-41 MuiTypography-body1-50 Header-headerText-40"
  >
    GENERAL  
    <br />
    KNOWLEDGE_
  </p>
</div>
`;
